pkg/version/version.go

```diff
@@ -43,6 +43,9 @@ type Versions struct {
 	// or rely on defaults.ImageFlavor if you need a default collector image tag.
 	ScannerVersion string `json:"ScannerVersion"`
 	ChartVersion   string `json:"ChartVersion"`
+	// The Database versioning needs to be added by the caller due to scoping issues of config availabilty
+	Database        string `json:"Database,omitempty"`
+	DatabaseVersion string `json:"DatabaseVersion,omitempty"`
```

Comment:
Database version is frequently used as the internal database version sequence number. Can we rename it to DatabaseServerVersion?